Patch by Zarafa <http://www.zarafa.com/> - this patch adds some characters that should be "escaped" in
quoted-printable. With this fix you can have these characters in a fullname of an email address.

--- libvmime-0.7.1/src/encoderQP.cpp					2005-03-18 22:26:47.000000000 +0100
+++ libvmime-0.7.1/src/encoderQP.cpp.quoted-printable-specials		2007-02-02 10:09:26.273653250 +0100
@@ -214,10 +214,18 @@
 			QP_ENCODE_HEX('=')
 			break;
 		}
+		// http://www.faqs.org/rfcs/rfc2047.html, see 'especials' characters
 		case ',':
 		case ';':
 		case ':':
 		case '_':
+		case '@':
+		case '(':
+		case ')':
+		case '<':
+		case '>':
+		case '[':
+		case ']':
 		{
 			if (rfc2047)
 			{
